src/StdHelper/include/WalkingControllers/StdHelper/Helper.h

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file Helper.h
+ * @authors Giulio Romualdi <giulio.romualdi@iit.it>
+ * @copyright 2018 iCub Facility - Istituto Italiano di Tecnologia
+ *            Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2018
+ */
+
+#ifndef WALKING_CONTROLLERS_STD_HELPER_H
+#define WALKING_CONTROLLERS_STD_HELPER_H
+
+// std
+#include <vector>
+#include <deque>
+
+/**
+ * Helper for std library
+ */
+namespace StdHelper
```

Comment:
What about defining a namespace ``WalkingControllers``  enclosing all these libraries? So that it would be ``WalkingControllers::StdHelper``.

Reply:
Here `StdUtilities`?